app/Domain/Mail/MailManager.php

```diff
@@ -37,14 +39,35 @@ public function sendIntegrationActivatedMail(IntegrationActivated $integrationAc
                     'lastName' => $contact->lastName,
                     'contactType' => $contact->type->value,
                     'integrationName' => $integration->name,
-                    //@todo Should we add a language preference to contacts?
                     'url' => $this->baseUrl . '/nl/integraties/' . $integration->id,
                     'type' => $integration->type->value,
                 ]
             );
         }
     }
 
+    public function sendIntegrationBlockedMail(IntegrationBlocked $integrationBlocked): void
```

Comment:
Is the direct URL of the integration inside `publiq-platform` useful for every type of mail?

Reply:
Difficult to say because we don't have a final copy of these emails.
Currently I am just using it for the activation email, but we will have to see how it ends up.
Would you prefer to always send all variables?